spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/ContractDownloader.java

```diff
@@ -70,14 +70,43 @@ public ContractDownloader(StubDownloader stubDownloader,
 	 * pattern
 	 * @return location of the unpacked downloaded stubs
 	 */
+	// Use unpackAndDownloadContracts() and createNewInclusionProperties() instead
+	@Deprecated
```

Comment:
These two methods are modifying `ContractVerifierConfigProperties`, so I deprecated them in favour of more straightforward ones (tow separate calls).